packages/shared/providers.yaml

```diff
@@ -219,6 +219,17 @@ epic-games:
     authorization_url: https://www.epicgames.com/id/authorize
     token_url: https://api.epicgames.dev/epic/oauth/v1/token
     authorization_method: header
+exact-online:
```

Comment:
This looks good! 

But you need to add a page to the documentation as well -> https://docs.nango.dev/contribute#step-3-add-your-integration-to-the-documentation

Reply:
whoops, because of all the issues with the API I forgot this. Will add it when I am at my pc.